### PR TITLE
Update the layout for the main feature and add the logout floating bu…

### PR DESCRIPTION
**Pull request by chaterinamf**
Did you write good & comprehensive specs? ⭐️
no

What's this PR do ? ⭐️
update the layout that matches to raendy's design and add the logout floating button

What are the relevant tickets ? ⭐️
https://www.notion.so/prawito/Create-layout-for-clock-in-clock-out-feature-in-mobile-773c3ee55062440987f44137822aa49d

Where should the reviewer start ?

How should this be manually tested ?

What is the Dockup URL ?

Any background context you want to provide ?

Any deployment note (migrations, new envs, rake tasks, cron) ?

Screenshots (if appropriate)
image

Definition of Done:

[x] Specs / Tests are adequate ? ⭐️

[x] Changes conform with code quality tools (Rubocop, Reek, Brakeman) ? ⭐️

[x] Does this PR require Database changes ?

[x] Does this PR require monitoring after deployment (Scout, logs, Sentry, etc) ?

[x] Does this PR require a regression test by QA ? All fixes require a regression test.

[x] Does this add new dependencies ? (network setup, new 3rd party services)

### CODE REVIEW

**Review by andranugraha — APPROVED** [review]

**Review by rahmatullah5 — CHANGES_REQUESTED** [review]
Please make sure all the image assets is already optimized, you can use lossless image compression such as https://kraken.io/

File: app/src/main/java/com/example/attendance/ui/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        clockInBtn = findViewById(R.id.clockIn_Btn)
+        clockOutBtn = findViewById(R.id.clockOut_Btn)
+        timeLog = findViewById(R.id.timeLogRV)
+
+        timeShow.setText(getCurrentTime())
+        scheduleDate.setText("${getDay()}, ${getTodayState()}")
+    }
+
+    fun getCurrentTime(): String {
+        val currentTime = SimpleDateFormat("hh:mm a", Locale.getDefault()).format(Date())
```
Comment (rahmatullah5) [inline app/src/main/java/com/example/attendance/ui/MainActivity.kt:39]: Why do we need to store the value first? can we just return it right `away?`

```
fun getCurrentTime(): String {
return SimpleDateFormat("hh:mm a", Locale.getDefault()).format(Date())
```

**Review by rahmatullah5 — COMMENTED** [review]
Please remove any comment if you will not use the code anymore

File: app/src/main/java/com/example/attendance/ui/MainActivity.kt (hunk 1 above)
Comment (chaterinamf) [reply app/src/main/java/com/example/attendance/ui/MainActivity.kt:39]: yes, it can. I store the value first just to make a clean appearance

**Review by rahmatullah5 — APPROVED** [review]

**Review by rahmatullah5 — APPROVED** [review]

**Pull request merged**